README.md

```diff
@@ -20,8 +20,16 @@ The above example will output coverage metrics for `foo.js`.
 
 run `c8 report` to regenerate reports after `c8` has already been run.
 
-## Disclaimer
+## Supported Node.js Versions
 
 c8 uses
 [bleeding edge Node.js features](https://github.com/nodejs/node/pull/22527),
 make sure you're running Node.js `>= 10.10.0`.
+
+## Goals of Project
```

Comment:
nit: Goals of `this` Project or Goals of `the` Project.

Reply:
On it 👍